photutils/aperture/core.py

```diff
@@ -108,13 +108,76 @@ class PixelAperture(Aperture):
     optionally, ``area``).
     """
 
+    def _prepare_plot(self, origin=(0, 0), source_id=None, ax=None,
+                      fill=False, **kwargs):
+        """
+        Prepare to plot the aperture(s) on a matplotlib Axes instance.
+
+        Parameters
+        ----------
+        origin : array-like, optional
+            The ``(x, y)`` position of the origin of the displayed
+            image.
+
```

Comment:
Would you like to add e.g. `where ``(x,y)`` corresponds to the usual rectangular coordinate system`? (if that's the case).